db/controllers/news-controllers.js

```diff
@@ -27,4 +31,15 @@ const getArticle = (request, response, next) => {
     });
 };
 
-module.exports = { getTopics, getEndpoints, getArticle };
+const getComments = (request, response, next) => {
+  const { article_id } = request.params;
```

Comment:
You will need to check the reason for the query returning empty rows:
- article exists but has no comments.
- article does not exist.
You will need to check whether the article exists in order to determine this.
I recommend the use of 'Promise.all()' here.